src/plotter/canvas.py

```diff
@@ -67,6 +67,8 @@ def save(self, path: str):
         oldIgnore = ROOT.gErrorIgnoreLevel
         ROOT.gErrorIgnoreLevel = 3000
         self.tcan.SaveAs(path)
+        if verbose:
```

Comment:
log.debug(path)

Reply:
Debug prints lots of useless crap atm. I want to be able to fetch a path of the plot I am working on. We can make mostly have verbosity False in bulk plotting